.gitmodules

```diff
@@ -18,3 +18,6 @@
 	url = https://github.com/libusb/libusb.git
 	branch = master
 	shallow = true
+[submodule "Externals/spirv_cross/SPIRV-Cross"]
+	path = Externals/spirv_cross/SPIRV-Cross
+	url = https://github.com/KhronosGroup/SPIRV-Cross.git
```

Comment:
This should probably also have

```
	branch = master
	shallow = true
```

like the other submodules.  (I'm not 100% sure what `branch` changes, but setting `shallow` reduces the size (or at least reduces the size until you manually pull it?).)

Reply:
https://www.git-scm.com/docs/gitmodules says that `branch` affects which branch is tracked, and it defaults to the remote `HEAD` otherwise.  I guess this would make a difference if they changed the default branch, or if they renamed `master` to `main`, but I'm not sure if that's the behavior we want.  Either way it's probably best to include it for consistency.